index.html

```diff
@@ -0,0 +1,298 @@
+            </span>
+            <span class="display-2--description">
+              SECOND EDITION<br> 
+              OF THE FITNESS<br> 
+              CAMPAIGN
+            </span>
+          </h1>
+        </div>
+        <div class="col-md-6-intros">
+          <div class="mt-3 mb-3">
+            <div class="alert alert-secondary w-50 p-3 border border-warning shadow-sm" role="alert">
+              When you're at the gym feeling like you'll never be one of those people who look like they've 
+              been at it their entire lives, remember that they all started somewhere.
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6-intros">
+          <h1 class="display-2">
+            <span class="display-2--schedule">
+              (MON) ~ (FRI)
+            </span>
+            <span class="display-2--locale">
+              @ National Fitness Academy
+            </span>
+          </h1>
+        </div>
+
+      </div>
+    </div>
+  </section>
+
+  <!--########################################
+               MAIN PROGRAM SECTION
+    #############################################-->
+  <section class="main-program">
+    <div class="container d-flex justify-content-center">
+      <div class="row text-center d-flex mt-3">
+        <h4 class="fw-bold lead mb-3 text-white">MAIN PROGRAM</h4>
+        <div class="heading-line mb-5"></div>
+      </div>
+    </div>
+    <div class="container d-flex">
+      <div class="row d-flex justify-content-center">
+        <div class="col-md-4 col-lg-2">
+          <div class="main-program__program-box shadow-sm">
+            <img src="images/arts/Fitness_Flatline.png" alt="fitness">
+          </div>
+        </div>
+        <div class="col-md-4 col-lg-2">
+          <div class="main-program__program-box shadow-sm">
+            <img src="images/arts/Basketball game_Flatline.png" alt="fitness">
+          </div>
+        </div>
+        <div class="col-md-4 col-lg-2">
+          <div class="main-program__program-box shadow-sm">
+            <img src="images/arts/Football team_Monochromatic.png" alt="fitness">
+          </div>
+        </div>
+        <div class="col-md-4 col-lg-2">
+          <div class="main-program__program-box shadow-sm">
+            <img src="images/arts/Hiking_Two Color.png" alt="fitness">
+          </div>
+        </div>
+        <div class="col-md-4 col-lg-2">
+          <div class="main-program__program-box shadow-sm">
+            <img src="images/arts/Hockey match_Isometric.png" alt="fitness">
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row text-center d-flex justify-content-center">
+      <h4 class="fw-bold lead mb-3 text-white"><ins>SEE THE WHOLE PROGRAM</ins></h4>
+    </div>
+    </div>
+  </section>
+  <!--###############################################
+              ACTIVITIES SECTION
+    #############################################-->
+  <section class="activities">
```

Comment:
- [ ] Please fix the section activities.
-
![1](https://user-images.githubusercontent.com/63972670/176970790-3953c9c2-3da5-434b-b78d-d3e82bb2a128.PNG)
 

- [ ] Please Make sure this section looks like the original design and in two colomn 

-
![1](https://user-images.githubusercontent.com/63972670/176971073-0168adc5-ba44-416c-bf32-a13fb9390f27.PNG)

- [ ] Mobile size: Something similar

![1](https://user-images.githubusercontent.com/63972670/176971282-bf09f18f-2a60-4c64-95d5-d100170baece.PNG)

- [ ] Please Open this link[ Original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and follow the design

Reply:
Done